### DISCUSSION TITLE
Can I change ADMINISTERED BY account for the instance I am running?

### DISCUSSION THREAD

**Discussion by janedzumerko** [mastodon/mastodon#21337] [Q&A] [answered]
I created an instance and I created an admin account for that one, which I don't use it. Meanwhile, I notice that every instance is administrated by an account from a user, so my question is - Can I change **ADMINISTERED BY** account for the instance I am running?

**Comment by supertassu** [comment, marked as answer]
Yes, this is the "Contact username" setting that can be found under "Administration" -> "Server settings" -> "Branding".

**Reply by janedzumerko** [reply]
Great, thanks! I will just add that it takes some time before the change is visible. At least this was my case, so I thought something was not working.
